app-frontend/src/store/reducers/__tests__/authorization.test.ts

```diff
@@ -0,0 +1,118 @@
+import authorizeUser, { initialState } from '../authorization';
+import signupUser, { signupUserFailed, signupUserSuccess } from '../../actions/authorizeUser/signup';
+import loginUser, { loginUserFailed, loginUserSuccess } from '../../actions/authorizeUser/login';
+import logoutUser from '../../actions/authorizeUser/logout';
+import getCurrentUser, { getCurrentUserFailed, getCurrentUserSuccess } from '../../actions/authorizeUser/getCurrentUser';
+import { userData } from '../../../mocks/store/constants';
+
+const user = {
+  email: '1@gmail.com',
+  password: '111111',
+};
+
+const registerUser = {
```

Comment:
I don't see any need for extracting values into header.
You can leave it here only in case of you need to use it across multiple tests.